public/pages/alarms/alarms.html

```diff
@@ -35,10 +35,13 @@ <h4>Expand time range</h4>
           Click the button below to open the time picker. For future reference you can open the time picker by clicking on the <a class="btn btn-xs navbtn" ng-click="kbnTopNav.toggle('filter')" aria-expanded="kbnTopNav.is('filter')" aria-label="time picker" data-test-subj="discoverNoResultsTimefilter"><i aria-hidden="true" class="fa fa-clock-o"></i> time picker</a> button in the top right corner of your screen.</p>
         </div>
       </div>
-      <label ng-repeat="col in alarm_columns">
-        {{col.label}}
-        <input type="checkbox" ng-model="col.visible">
-      </label><br/>
+      <form class="form-inline" novalidate>
```

Comment:
Please add some right padding for the checkbox labels here https://github.com/sirensolutions/sentinl/blob/master/public/style/main.less#L164

For example:
```
  .container .form-inline .checkbox label {
    padding-right: 10px;
  }
```
![screenshot from 2018-07-12 18-07-36](https://user-images.githubusercontent.com/5389745/42646397-1f5b8cc8-8601-11e8-92bc-6fbde23e35ff.png)

Reply:
You can always check if CSS selectors work using Chrome devtools: https://developers.google.com/web/updates/2015/05/search-dom-tree-by-css-selector